canary/scripts/test_hmr.py

```diff
@@ -7,7 +7,7 @@
 from openff.toolkit.topology import Molecule
 from openmmforcefields.generators import SystemGenerator
 
-DATA_PATH = importlib.resources.files("canary") / "data"
+DATA_PATH = pathlib.Path(".") / "canary" / "data"
```

Comment:
I'm surprised this ever worked; there was never a `canary/__init__.py` file or anything that could be used to install it.